client/src/components/CreateCommunity/index.js

```diff
@@ -93,7 +93,6 @@ export default class CreateCommunity extends Component {
           rc_uid: Cookies.get("rc_uid"),
           channel: community.value.concat("_community"),
           members: communityMembers,
-          description: description,
```

Comment:
In case a user has multiple organizations(apart from user org), then creating a community for those orgs will require a description to be set from the client-side.

Reply:
@djcruz93 this is **already** being done for both user and non-user communities over [here](https://github.com/murtaza98/RC4Git/blob/9a4b05d6d27db905476b3c5ef587dd75db232f18/client/src/components/CreateCommunity/index.js#L112-L124) :)

The above change is a refactor. We no longer need the `description` in the body since we're setting description for all kinds of communities (and channels) completely client-side.

Reply:
This type of API design is too confusing to understand, we should move create channels and topics to client-side asap.